rust/src/lib.rs

```diff
@@ -565,32 +555,87 @@ pub extern "C" fn zcashlc_init_blocks_table(
     unwrap_exc_or_null(res)
 }
 
-/// Returns the address for the account.
+/// Returns the most-recently-generated unified payment address for the specified account.
 ///
-/// Call `zcashlc_string_free` on the returned pointer when you are finished with it.
+/// # Safety
+///
+/// - `db_data` must be non-null and valid for reads for `db_data_len` bytes, and it must have an
+///   alignment of `1`. Its contents must be a string representing a valid system path in the
+///   operating system's preferred representation.
+/// - The memory referenced by `db_data` must not be mutated for the duration of the function call.
+/// - The total size `db_data_len` must be no larger than `isize::MAX`. See the safety
+///   documentation of pointer::offset.
+/// - Call [`zcashlc_string_free`] to free the memory associated with the returned pointer
+///   when done using it.
 #[no_mangle]
-pub extern "C" fn zcashlc_get_address(
+pub extern "C" fn zcashlc_get_current_address(
```

Comment:
```suggestion
pub unsafe extern "C" fn zcashlc_get_current_address(
```

Reply:
Deferring this to https://github.com/zcash-hackworks/zcash-light-client-ffi/issues/65#issuecomment-1315988668.